cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -0,0 +1,143 @@
+| `CREATE UNIQUE`     | Clause | Removed | Running such queries will fall back to using Cypher 3.1 (and use the rule planner)
+| `START`     | Clause | Removed | Running such queries will fall back to using Cypher 3.1 (and use the rule planner)
+| `MATCH (n)-[rs*]-() RETURN rs`     | Syntax | Deprecated | Replaced by `MATCH p=(n)-[*]-() RETURN relationships(p) AS rs`
+| `MATCH (n)-[:A\|:B\|:C {foo: 'bar'}]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[:A\|B\|C {foo: 'bar'}]-() RETURN n`
+| `MATCH (n)-[x:A\|:B\|:C]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[x:A\|B\|C]-() RETURN n`
+| `MATCH (n)-[x:A\|:B\|:C*]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[x:A\|B\|C*]-() RETURN n`
+| <<user-defined-aggregation-functions, User-defined aggregation functions>> | Functionality | Added |
+| <<query-schema-index-introduction, Composite indexes>> | Index | Added |
+| <<query-constraint-node-key, Node Key>> | Index | Added | Neo4j Enterprise Edition only
+| `CYPHER runtime=compiled` (Compiled runtime) | Functionality | Added | Neo4j Enterprise Edition only
+| <<functions-reverse-list,reverse()>> | Function  | Extended | Now also allows a list as input
+| <<functions-max, max()>>, <<functions-min, min()>> | Function  | Extended | Now also supports aggregation over a set containing both strings and numbers
+|===
+
+[[cypher-deprecations-additions-removals-3.3]]
+=== Version 3.3
+[options="header"]
+|===
+| Feature          | Type | Change | Details
+| `START`     | Clause | Removed | Replaced by the built-in procedures `db.nodeManualIndexSeek`, `db.nodeManualIndexSearch`, `db.relationshipManualIndexSeek`, and `db.relationshipManualIndexSearch`
+| `CYPHER runtime=slotted` (Faster interpreted runtime) | Functionality | Added | Neo4j Enterprise Edition only
+| <<functions-max, max()>>, <<functions-min, min()>> | Function  | Extended | Now also supports aggregation over a set containing lists of strings and numbers
+|===
+
+[[cypher-compatibility]]
+== Compatibility
+
+Older versions of the language can still be accessed if required.
+There are two ways to select which version to use in queries.
+
+. Setting a version for all queries
+You can configure your database with the configuration parameter `cypher.default_language_version`, and enter which version you'd like to use (see <<cypher-versions>>).
+Every Cypher query will use this version, provided the query hasn't explicitly been configured as described in the next item below.
+
+. Setting a version on a query by query basis
+The other way is on a query by query basis.
+By simply putting `CYPHER 2.3` at the beginning, that particular query will be executed with the version of Cypher included in Neo4j 2.3.
+
+Below is an example using the `START` clause to access a legacy index:
+
+[source, cypher]
+----
+CYPHER 2.3
+START n=node:nodes(name = "A")
+RETURN n
+----
+
+
+[[cypher-start-compatibility]]
+=== Accessing entities by id via START
```

Comment:
This entire section seems like content from a Cypher 2.2 or Cypher 2.3 manual. Everything above here is very much about Neo4j 3.0 and later. Do we still need this information about accessing nodes by id using START at all?

Reply:
You're right - esp. in light of the new Deprec/Adds/Rems table, which explains the gist of this anyway. Removing in 3.1, 3.2 and 3.3

Reply:
👍